File: js.v/js/death.js
function die(personId, cause) {
    console.log(`Person \#${personId} DIED because of ${cause}.`);
    world[personId].alive = false;
}

function checkSurvivor() {
    if (countAlive() === 1) {
        refreshStat();
        let survivedGuy;
        for (var i = 0; i < peopleC; i++) {
            if (world[i].alive) { survivedGuy = i; break;}
        }
        alert("Person \#" + i + " is the only SURVIVOR from this world!!!");
        return true;
    }
    else {
        return false;
    }
}

File: src/death.py
from config import *

def die(wid, cause):
    print('Person #%s: DIED because of %s.' % (wid, cause))
    world[wid].alive = False

# def checkSurvivor():
#     if countAlive() == 1:
#         survivedGuy = None
#         for i in range(0, poepleC):
#             if world[i].alive:
#             	survivedGuy = i
#             	break
#         print("Person #" + i + " is the only SURVIVOR from this world!!!")
#         return true
#     else:
#         return false
